packages/mongodb-chatbot-server/src/routes/responses/responsesRouter.ts

```diff
@@ -0,0 +1,40 @@
+import Router from "express-promise-router";
+import validateRequestSchema from "../../middleware/validateRequestSchema";
+import {
+  makeCreateResponseRoute,
+  CreateResponseRequest,
+} from "./createResponse";
+
+interface ResponsesService {
+  generateResponse: () => void;
+}
+
+export interface ResponsesRouterParams {
+  responses: ResponsesService;
+  supportedModels: string[];
+  maxOutputTokens: number;
+}
```

Comment:
can u do this nested, so its like:

```suggestion
export interface ResponsesRouterParams {
  createResponse: {
    responses: ResponsesService;
    supportedModels: string[];
    maxOutputTokens: number;
  }
}
```

this way when we add other config to the router, it'll be clearer whats going on